### PR TITLE
Small update batch

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: bootstrap.d/dev-libs.yml [hunk 1]
```diff
@@ -1606,11 +1606,18 @@ packages:
   - name: weston
     labels: [aarch64]
     architecture: '@OPTION:arch@'
+    metadata:
+      summary: 'A lightweight and functional Wayland compositor'
+      description: 'Weston is a Wayland compositor designed for correctness, reliability, predictability, and performance. Out of the box, Weston provides a very basic desktop, or a full-featured environment for non-desktop uses such as automotive, embedded, in-flight, industrial, kiosks, set-top boxes and TVs.'
+      spdx: 'MIT'
+      website: 'https://gitlab.freedesktop.org/wayland/weston'
+      maintainer: "no92 <no92.mail@gmail.com>"
```
Comment (Dennisbonke) [inline bootstrap.d/dev-libs.yml:1614]: Use your @managarm.org address here too.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
One comment

File: bootstrap.d/dev-libs.yml (hunk 1 above)
Comment (no92) [reply bootstrap.d/dev-libs.yml]: Done.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**